blazor/dropdown-list/value-binding.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: Value Binding in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Value Binding in Syncfusion Blazor DropDown List component and more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Value Binding in Dropdown List
+
+The value binding as the process of passing values between a component and its parent. There are two methods for binding values, as shown below.
+
+    * bind-Value Binding 
+    * bind-Index Binding
+
+## Bind value binding
+
+The value binding can be achieved by using [`@bind-Value`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute and it supports string, int, Enum, bool and complex types. If component value has been changed, it will affect all places where you bind the variable for the [`@bind-value`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute.

```

Comment:
The value binding can be achieved by using the [`@bind-Value`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute, and it supports string, int, Enum, bool, and complex types. If the component value has been changed, it will affect all places where you bind the variable for the [`@bind-value`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute.